### PR TITLE
Add overloaded shortcut method to register BeanDefinition

### PR DESCRIPTION
**Pull request by liaozan**

### CODE REVIEW

**Comment by liaozan** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by mghio — CHANGES_REQUESTED** [review]
Please add some unit test in [BeanRegistrationUtilTest](https://github.com/apolloconfig/apollo/blob/master/apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java).

**Comment by liaozan** [comment]
Updated

File: apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java [hunk 1]
```diff
@@ -68,7 +66,12 @@ public void registerBeanDefinitionIfNotExistsWithExtPropTest() {
     extraPropertyValues.put(someBeanName, "someProperty");
     assertTrue(BeanRegistrationUtil.registerBeanDefinitionIfNotExists(someRegistry, someBeanName,
         getClass(), extraPropertyValues));
+  }
 
+  @Test
+  public void registerBeanDefinitionIfNotExistsWithoutBeanNameTest() {
+    someRegistry.registerBeanDefinition(BeanDefinition.class.getName(), Mockito.mock(BeanDefinition.class));
+    assertFalse(BeanRegistrationUtil.registerBeanDefinitionIfNotExists(someRegistry, BeanDefinition.class));
```
Comment (nobodyiam) [inline apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java:74]: It's better to add a unit test to cover `public static boolean registerBeanDefinitionIfNotExists(BeanDefinitionRegistry registry, Class<?> beanClass, Map<String, Object> extraPropertyValues)`.

Comment (liaozan) [reply apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java:73]: added

**Review by mghio — APPROVED** [review]
LGTM

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**